templates/narrativeqa/templates.yaml

```diff
@@ -0,0 +1,109 @@
+dataset: narrativeqa
+templates:
+  3b53c95c-022b-4a51-946a-6c88b962892a: !Template
+    id: 3b53c95c-022b-4a51-946a-6c88b962892a
+    jinja: 'Synopsis: {{ document.summary.text }}
+
+
+      Answer the question.
+
+
+      {{ question.text }}
+
+
+      Full text: {{ document.text }} |||
+
+      {% set i= range(0, answers|length) | random %}
+
+      {{ answers[i].text }}'
+    name: Template_05
+    reference: Text + Summary
+    task_template: false
+  62df1726-be83-4403-9483-732da1174cf7: !Template
+    id: 62df1726-be83-4403-9483-732da1174cf7
+    jinja: '{{ document.text }}
+
+
+      Using the above text, answer the following question.
+
+
+      {{ question.text }} |||
+
+      {% set i= range(0, answers|length) | random %}
+
+      {{ answers[i].text }}'
+    name: Template_01
+    reference: Text only
+    task_template: true
+  87cb2e86-0764-412a-ba2d-fe3172997a25: !Template
+    id: 87cb2e86-0764-412a-ba2d-fe3172997a25
+    jinja: '{{ document.text }}
+
+
+      Summarize the given document. |||
+
+      {{ document.summary.text }}'
+    name: Template_06
+    reference: Text summarization
+    task_template: false
```

Comment:
note: this one will be particularly hard since we'll have to truncate very aggressively the `document.text` because of its absurd long length

Reply:
Okay, should I remove templates 06-08 then?

Reply:
i think it's fine, i prefer handling these in the truncation logic, just wanted to flag this problem